ProcessLib/ThermoMechanics/ThermoMechanicsProcessData.h

```diff
@@ -0,0 +1,80 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+namespace MeshLib
+{
+class Element;
+}
+
+namespace ProcessLib
+{
+namespace ThermoMechanics
+{
+template <int DisplacementDim>
+struct ThermoMechanicsProcessData
+{
+    ThermoMechanicsProcessData(
+        std::unique_ptr<MaterialLib::Solids::MechanicsBase<DisplacementDim>>&&
+            material_,
+        Parameter<double> const& solid_density_,
+        Parameter<double> const& linear_thermal_expansion_coefficient_,
+        Parameter<double> const& specific_heat_capacity_,
+        Parameter<double> const& thermal_conductivity_,
+        double const reference_temperature_,
+        Eigen::Matrix<double, DisplacementDim, 1> const& specific_body_force_)
+        : material{std::move(material_)},
+          solid_density(solid_density_),
+          linear_thermal_expansion_coefficient(
+              linear_thermal_expansion_coefficient_),
+          specific_heat_capacity(specific_heat_capacity_),
+          thermal_conductivity(thermal_conductivity_),
+          reference_temperature(reference_temperature_),
+          specific_body_force(specific_body_force_)
+    {
+    }
+
+    ThermoMechanicsProcessData(ThermoMechanicsProcessData&& other)
+        : material{std::move(other.material)},
+          solid_density(other.solid_density),
```

Comment:
I say this here, but it refers to all instances: solid_reference_density (see previous comments)